src/io/openshift/plugins/analytics.groovy

```diff
@@ -1,10 +1,29 @@
 package io.openshift.plugins
 
 import io.openshift.Events
+import io.openshift.Utils
 
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
```

Comment:
the url and sha can be passed as arguments here so that you can simply access those as 
`args.git_url`, `args.git_sha` 

cc: @hrishin @piyush-garg

Reply:
@sthaha make sense. Will send the PR for this.

Reply:
@yzainee now you can access git URL and commit id with `events.git` https://github.com/fabric8io/osio-pipeline/pull/49/files#diff-5ee6d7efbcc2ff1a67ee14a2db6b1221R39

Would you mind using that?

Reply:
@yzainee do you want to make changes or shall I merge it?

Reply:
@hrishin Please go ahead and merge this

Reply:
@yzainee could you remove the log line?

Reply:
@yzainee do we want to print this line? if Yes then could you please remove the debug variables `$e $a`  from `echo` statement? It would not look good to dump those values build log